packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/RecordHistoryEvents/index.js

```diff
@@ -0,0 +1,94 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+
+import formParts from 'coreModules/form/components/parts'
+import { withI18n } from 'coreModules/i18n/higherOrderComponents'
+
+import EventRow from './EventRow'
+
+const { TranslatedHeader } = formParts
+
+const mapStateToProps = (state, { formValueSelector, name }) => {
+  const recordHistoryEvents = formValueSelector(state, name)
+
+  if (!recordHistoryEvents || !recordHistoryEvents.length) {
+    return {}
+  }
+
+  return {
+    createdEvent: recordHistoryEvents.find(({ action }) => action === 'create'),
+    lastModifiedEvent: recordHistoryEvents.find(
+      ({ action }) => action === 'update'
+    ),
+  }
+}
+
+const propTypes = {
+  createdEvent: PropTypes.object,
+  i18n: PropTypes.shape({
+    moduleTranslate: PropTypes.func.isRequired,
+  }).isRequired,
+  lastModifiedEvent: PropTypes.object,
```

Comment:
the two event should have PropTypes.shape with the attributes specified because they are used in here